### ISSUE TITLE
User Should Be Able to Use Tab to Access the Skip to Content Link (via Screenreader)

### ISSUE THREAD

**Issue by openstaxalina** [openstax/webview#2196]
Report routed via Nicole:
In testing the site with NVDA, I initially was not successful in located the Skip to Content link. While following the conversation, I had an idea and decided to go recheck. 
I was able to find the Skip to Content link by utilizing Shift+Tab (like tabbing through, but backwards). I'm not sure why it required this backwards navigation... but it could be a bit confusing for users.

**Comment by openstaxalina** [comment]
@philschatz diagnosed this - this occurs because CNX is a single-page app:

1 - From any given page, find and select the Skip to Content option
2 - Once the new page loads, notice that your previous page position was "saved", so when you tab forward, you have already passed the Skip to Content option

**Comment by openstaxalina** [comment]
@Stackblocks to also test whether the screenreader speaks, "Skip to Content".

**Comment by openstaxalina** [comment]
Per @edwoodward, this is likely resolved by setting the tab index to 0 on each page.

**Comment by brittany-johnson** [comment]
@Stackblocks Ready for testing!

**Comment by Stackblocks** [comment]
@brittany-johnson It looks like activating the skip to content button acts more as a sort of page-down, than a jump to the start of the content. When it is first used it blows past the start of the body content and when interacted with again it doesn't go back to the same spot, it does another page-down-esque hop. 

The button also only works when the space bar is pressed, not when return is pressed.

## Tested Content:
https://easyvm5.cnx.org/contents/kUrGbuHs@5.1:6KQQIO6W@3/Ethics-and-Profitability

## Repro steps: 
1. Target the URL bar by clicking in it / using F6 / Using CMD + L
2. Press Tab once to hop to the jump button
3. Press the space bar

**Comment by brittany-johnson** [comment]
Hmmm I tested this fix by pressing the return key on the skip to content link and not the space bar key locally and I was able to do it again just now by following your steps but replacing step 3 with return key instead of space bar key.  
I didn't add any code to alter the native functionality of the space bar key and at least when I tried it just now, the space bar key seems to be working as expecting, navigating from one chunk of the page to another. 
I may be misunderstanding you though, so we can talk more tomorrow

**Comment by Stackblocks** [comment]
Turns out Page-Down is just the "press spacebar in the content" action, so on the plus side that isn't something that needs fixing. What that means, though, is that I was unable to get it to activate with either return or space so i'll test it again in the morning and failing that i'll re-deploy and try again. Dragging the issue back into QA column.

**Comment by Stackblocks** [comment]
Working correctly in Chrome, FireFox has some browser-side hangups about this kind of thing. 

```
Firefox works similarly, with the exception that on OS X a elements aren’t tab-accessible by default, with or without tabindex. 
You can access a elements with the keyboard in Firefox two ways. 
You can hold down the option key while pressing tab, or you can change your system preferences. 
Firefox respects an option called “Full Keyboard Access” in your keyboard preferences, which controls which types of controls can be accessed with tab on your computer. 
If you toggle that option to “All Controls”, Firefox will focus on links like nobody’s business.
```

Marking as resolved.

**Comment by Stackblocks** [comment]
Follow-up from a11y chat: it is likely that a11y users will already have this setting active and it should not present an issue. We've updated Customer Support with instructions on how to handle this if it comes up.
